<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KCodeeditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QsciScintilla" name="textEdit">
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>674</width>
          <height>100</height>
         </size>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>406</width>
           <height>106</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="window-close">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>find</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>regular expression</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>case sensitive</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string>whole word</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Replace</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionclose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionFind"/>
    <addaction name="actionDeselect"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuChange_Language">
     <property name="title">
      <string>La&amp;nguage</string>
     </property>
     <addaction name="actionNormal"/>
     <addaction name="actionBash"/>
     <addaction name="actionBatch"/>
     <addaction name="actionCmake"/>
     <addaction name="actionCoffeeScript"/>
     <addaction name="actionC"/>
     <addaction name="actionC_2"/>
     <addaction name="actionC_3"/>
     <addaction name="actionCSS"/>
     <addaction name="actionD"/>
     <addaction name="actionFortan"/>
     <addaction name="actionHTML"/>
     <addaction name="actionJSON"/>
     <addaction name="actionLua"/>
     <addaction name="actionMakeFIle"/>
     <addaction name="actionMarkDown"/>
     <addaction name="actionMat"/>
     <addaction name="actionPascal"/>
     <addaction name="actionPerl"/>
     <addaction name="actionPython"/>
     <addaction name="actionRuby"/>
     <addaction name="actionSQL"/>
     <addaction name="actionTeX"/>
     <addaction name="actionYAML"/>
     <addaction name="actionXML"/>
    </widget>
    <addaction name="actionWordwrap"/>
    <addaction name="actionEnlarge"/>
    <addaction name="actionShrink_Font"/>
    <addaction name="actionShow_line_numbers"/>
    <addaction name="menuChange_Language"/>
    <addaction name="actionChange_Font"/>
    <addaction name="actionchange_font_color"/>
    <addaction name="actionstatus_bar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;ve As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionWordwrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wordwrap</string>
   </property>
  </action>
  <action name="actionEnlarge">
   <property name="text">
    <string>&amp;Enlarge</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionShrink_Font">
   <property name="text">
    <string>&amp;Shrink Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionShow_line_numbers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;line numbers</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>&amp;Find and Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>R&amp;eplace</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>&amp;Normal</string>
   </property>
  </action>
  <action name="actionBash">
   <property name="text">
    <string>&amp;Bash</string>
   </property>
  </action>
  <action name="actionBatch">
   <property name="text">
    <string>Batch</string>
   </property>
  </action>
  <action name="actionCmake">
   <property name="text">
    <string>&amp;Cmake</string>
   </property>
  </action>
  <action name="actionCoffeeScript">
   <property name="text">
    <string>C&amp;offeeScript</string>
   </property>
  </action>
  <action name="actionC">
   <property name="text">
    <string>C</string>
   </property>
  </action>
  <action name="actionC_2">
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionC_3">
   <property name="text">
    <string>C#</string>
   </property>
  </action>
  <action name="actionCSS">
   <property name="text">
    <string>CSS</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>&amp;D</string>
   </property>
  </action>
  <action name="actionFortan">
   <property name="text">
    <string>&amp;Fortan</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="text">
    <string>&amp;HTML</string>
   </property>
  </action>
  <action name="actionJSON">
   <property name="text">
    <string>&amp;JSON</string>
   </property>
  </action>
  <action name="actionLua">
   <property name="text">
    <string>&amp;Lua</string>
   </property>
  </action>
  <action name="actionMakeFIle">
   <property name="text">
    <string>&amp;MakeFIle</string>
   </property>
  </action>
  <action name="actionMarkDown">
   <property name="text">
    <string>Mar&amp;kDown</string>
   </property>
  </action>
  <action name="actionMat">
   <property name="text">
    <string>Mat</string>
   </property>
  </action>
  <action name="actionPascal">
   <property name="text">
    <string>&amp;Pascal</string>
   </property>
  </action>
  <action name="actionPerl">
   <property name="text">
    <string>P&amp;erl</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionRuby">
   <property name="text">
    <string>&amp;Ruby</string>
   </property>
  </action>
  <action name="actionSQL">
   <property name="text">
    <string>&amp;SQL</string>
   </property>
  </action>
  <action name="actionTeX">
   <property name="text">
    <string>&amp;TeX</string>
   </property>
  </action>
  <action name="actionYAML">
   <property name="text">
    <string>&amp;YAML</string>
   </property>
  </action>
  <action name="actionXML">
   <property name="text">
    <string>&amp;XML</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionChange_Font">
   <property name="text">
    <string>&amp;Change Font</string>
   </property>
  </action>
  <action name="actionchange_font_color">
   <property name="text">
    <string>Change &amp;Font Color</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="icon">
    <iconset theme="document-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionDeselect">
   <property name="text">
    <string>&amp;Deselect</string>
   </property>
  </action>
  <action name="actionstatus_bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show status &amp;bar</string>
   </property>
  </action>
  <action name="actionReplace_2">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_All</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShrink_Font</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnlarge</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
